frontend/src/modules/topics/topic.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Row, Col, Card, Image, Button } from "antd";
+import { LoadingOutlined } from "@ant-design/icons";
+
+import "./styles.scss";
+import imageNotFound from "../../images/image-not-found.png";
+import TopicImages from "./topic-images";
+
+import { UIStore } from "../../store";
+import isEmpty from "lodash/isEmpty";
+
+const Topic = () => {
+  const tags = UIStore.useState((s) => s.tags);
+
+  const isLoaded = () => Boolean(!isEmpty(tags));
+
+  return (
+    <div id="topics">
+      {/* Featured topic */}
+      <div className="section-featured-topic-container">
+        <div className="ui container">
+          <div className="section-container">
+            <h2>Featured Topics</h2>
+            <Row className="topic-item-wrapper" gutter={[16, 16]}>
+              {!isLoaded() || !tags?.topics ? (
+                <h2 className="loading">
+                  <LoadingOutlined spin /> Loading...
+                </h2>
+              ) : (
+                renderTopics(tags.topics)
+              )}
+            </Row>
+          </div>
+        </div>
+      </div>
+      {/* All topic */}
+      <div className="section-all-topic-container"></div>
+    </div>
+  );
+};
+
+const renderTopics = (topics) => {
+  return topics.map((x) => {
+    const { id, tag } = x;
+    const image = TopicImages.find((x) => x.tag === tag).image;
+
+    return (
+      <Col key={`${tag}-${id}`} sm={24} md={12} lg={8}>
+        <Link
+          to={{
+            pathname: "/browse",
+            search: `?tag=${tag}`,
+          }}
+        >
+          <Card className="topic-item-card">
+            <span className="featured-topic-label resource-label">
+              Featured
+            </span>
+            <Image
+              className="topic-item-image"
+              width="100%"
+              src={image || imageNotFound}
+              preview={false}
+            />
+            <div className="topic-item-title-wrapper">
+              <div className="topic-item-title">{tag}</div>
+              <Button className="topic-item-count" shape="circle" type="ghost">
```

Comment:
I think we can get rid of the counts, for now. It is going to be tricky to get the counts before we - 1. make sure that Initiatives use the same tags as other resources (including IDs) and 2. we make sure that the topics names are not repeated in tags.  I'd say we get rid of the counts as part of the MVP.

Reply:
I'll comment the count code, for future development